plugins/talk-plugin-facebook-auth/README.md

```diff
@@ -27,24 +27,23 @@ You can learn more about getting a Facebook App ID at the
   their [Creating an App ID](https://developers.facebook.com/docs/apps/register)
   guide.
 
-**Setting up your FaceBook app:**
-* Go to My Apps > Create A New App
-* Add an App Name and Email to create an app id
-* Confirm that you are not a robot
-* Go to Settings > Basic:
+**Setting up your Facebook app:**
+* Go to [Facebook Developers Portal](https://developers.facebook.com) and click on Getting Started or My Apps
+* Create a new app > set the App Name and Email to create an app id
+* Confirm that you are not a robot, then configure the app as follows:
+* In Settings > Basic:
     * add app domains (your Talk domain)
     * add a link to your privacy policy
     * add a link to your terms of service
-    * save changes
-* Go to Settings > Advanced:
+* In Settings > Advanced:
     * turn on "Require App Secret"
-* Under "Products" click + to add a Product:
-    * Select "Let people log in with FaceBook"
+* Add a "Product" (Under "Products" click + to add a Product):
+    * Setup "Facebook Login"
     * choose `www` 
     * enter your Talk domain url
     * click _Next_ several times to get through the add code steps (You do not need to modify any code, the plugin takes care of this part for you.)
-* Go back to Settings > Basic:
-    * enter the callback url (Use your Talk domain with this endpoint: `/api/v1/auth/facebook/callback`)
+* Under Product Settings:
+    * set Valid OAuth Redirect URIs to your callback url (Use your Talk domain with this endpoint: `/api/v1/auth/facebook/callback`)
     * Locate your App Id and App Secret, set these as config vars on your instance of Talk
```

Comment:
Maybe add a line here: You can find both under `Settings -> Basic`

Reply:
This is the one that wasn't actually under Settings > Basic, it really is under the product settings, so was incorrect before, but is correct now.